Providers/Profile.swift

```diff
@@ -222,20 +205,6 @@ extension Profile {
     }
 }
 
-private let PrefKeyClientID = "PrefKeyClientID"
-extension Profile {
-    var clientID: String {
```

Comment:
This is good cleanup although its strange that this has not been used

Reply:
I haven't dug into the history but I'd be reasonably sure it was used at one point but was just never cleaned up.